views/chartTest.pug

```diff
@@ -24,14 +24,12 @@ block content
 
       var socket = io.connect('//'+document.location.hostname+':'+document.location.port);
       socket.on('chartData', function (data) {
-        console.log(data);
-
         window.chart.push([{
           time: data.time,
           y: data.value
```

Comment:
So for now hardcode this to be `y: data.device_data.sensor_data` but I think that we need to be more clever about this. 

i.e. Since there needs to be a separate chart for each of the members of the data object. We can make a new chart for each of them using `Object.keys(data)` and when new data comes in update the graph accordingly.